### PR TITLE
Did ele sham, druid fix(?)es

### PR DESCRIPTION
**Pull request by SovietFarmer**

### CODE REVIEW

File: behaviors/wow_retail/druid/restoration.lua [hunk 1]
```diff
@@ -150,6 +150,24 @@ local function DruidRestoDamage()
   end
 end
 
+local blacklist = {
+  [118] = "Polymorph",
+  [51514] = "Hex"
+}
+
+local function DruidPVPSaveMeHayZues()
```
Comment (ejt1) [inline behaviors/wow_retail/druid/restoration.lua:158]: Function name not descriptive

File: behaviors/wow_retail/druid/restoration.lua [hunk 2]
```diff
@@ -202,6 +224,27 @@ local function DruidRestoHeal()
...
   end
 
+  -- DO PVP SHEEEET
+  if Settings.DruidRestoPvPMode then
+    if Me.ShapeshiftForm == ShapeshiftForm.Normal then
+      local units = wector.Game.Units
+      for _, u in pairs(units) do
+        if Me:CanAttack(u) and u.IsPlayer then
+          --- this is a player that can be attacked, let's print his name to check
+          print(u.NameUnsafe)
```
Comment (ejt1) [inline behaviors/wow_retail/druid/restoration.lua:234]: Remove debug logging

File: behaviors/wow_retail/paladin/holy.lua [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+end
+
+local function DivineToll(lowest)
+  if not lowest or Spell.DivineToll:CooldownRemaining() > 0 then return end
+  local friends, count = Heal:GetMembersBelow(Settings.DivineTollPct)
+
+  return count > 2 and Spell.DivineToll:CastEx(lowest)
+end
+
+local function SeasonBlesings()
```
Comment (ejt1) [inline behaviors/wow_retail/paladin/holy.lua:260]: Mistyped blessings

File: system/heal.lua [hunk 4]
```diff
@@ -71,7 +76,7 @@ function Heal:WeighFilter()
         istank = true
       end
       if member.Role & GroupRole.Healer == GroupRole.Healer then priority = priority + 10 end
-      if member.Role & GroupRole.Damage == GroupRole.Damage then priority = priority + 5 end
+      if member.Role & GroupRole.Damage == GroupRole.Damage then priority = priority + 5 isdps = true end
```
Comment (ejt1) [inline system/heal.lua:79]: missing newline before `isdps = true`

**Review by ejt1 — CHANGES_REQUESTED** [review]

File: behaviors/wow_retail/paladin/holy.lua (hunk 3 above)
Comment (SovietFarmer) [reply behaviors/wow_retail/paladin/holy.lua:260]: Neer is ESL too.

**Review by ejt1 — APPROVED** [review]

**Pull request merged**